docs/src/preprocess/index.js

```diff
@@ -207,10 +197,15 @@ async function processMarkdownFilesInDir(rootDir, docsDir, regex) {
         matchesFound = true;
         const fullMatch = match[0];
         const identifier = match[1];
-        const codeFilePath = match[2]; // Absolute path to the code file from the root of the Docusaurus project
+        let codeFilePath = match[2];
         const language = match[3];
         const opts = match[4] || "";
 
+        if (codeFilePath.slice(0) != "/") {
+          // Absolute path to the code file from the root of the Docusaurus project
+          codeFilePath = `/${codeFilePath}`;
```

Comment:
```suggestion
          // Note: without prefixing with `/`, the later call to `path.resolve()` gives an incorrect path (absolute instead of relative)
          codeFilePath = `/${codeFilePath}`;
```